tests/expected/ptr_to_ref_cast/expected

```diff
@@ -0,0 +1,75 @@
+Checking harness check_zst_deref...
+
+Status: FAILURE
+Description: "dereferencing a pointer to invalid memory location"
+Location: tests/expected/ptr_to_ref_cast/ptr_to_ref_cast.rs:131:35 in function zst_deref
```

Comment:
We tend to avoid having line numbers in expected files as they are fragile, e.g. due to changing the file preamble. You can keep the function name though, e.g.
```suggestion
in function zst_deref
```
and add a backslash at the end of the previous line.